dasgo/gateway.go

```diff
@@ -132,31 +132,33 @@ type Command interface{}
 // Identify Structure
 // https://discord.com/developers/docs/topics/gateway#identify-identify-structure
 type Identify struct {
-	Token          string                       `json:"token,omitempty"`
-	Properties     IdentifyConnectionProperties `json:"properties,omitempty"`
-	Compress       bool                         `json:"compress,omitempty"`
+	Token          string                       `json:"token"`
+	Properties     IdentifyConnectionProperties `json:"properties"`
+	Compress       *bool                         `json:"compress,omitempty"`
 	LargeThreshold int                          `json:"large_threshold,omitempty"`
 	Shard          *[2]int                      `json:"shard,omitempty"`
 	Presence       GatewayPresenceUpdate        `json:"presence,omitempty"`
-	Intents        BitFlag                      `json:"intents,omitempty"`
+	Intents        BitFlag                      `json:"intents"`
 }
 
 // Identify Connection Properties
 // https://discord.com/developers/docs/topics/gateway#identify-identify-connection-properties
 type IdentifyConnectionProperties struct {
-	OS      string `json:"$os,omitempty"`
-	Browser string `json:"$browser,omitempty"`
-	Device  string `json:"$device,omitempty"`
+	OS      string `json:"$os"`
+	Browser string `json:"$browser"`
+	Device  string `json:"$device"`
 }
 
 // Resume Structure
 // https://discord.com/developers/docs/topics/gateway#resume-resume-structure
 type Resume struct {
-	Token     string `json:"token,omitempty"`
-	SessionID string `json:"session_id,omitempty"`
-	Seq       uint32 `json:"seq,omitempty"`
+	Token     string `json:"token"`
+	SessionID string `json:"session_id"`
+	Seq       uint32 `json:"seq"`
 }
 
+// TODO: was unclear
```

Comment:
Discord sources its documentation through high school labor. Basically, the heartbeat event will look like this:
```json
{
	"op": 1,
	"d": 251
}
```

It should be two `int` fields.